packages/core/src/types.ts

```diff
@@ -120,6 +120,7 @@ export type NormalizedOverrideOutput = {
   useNamedParameters?: boolean;
   enumGenerationType: EnumGeneration;
   suppressReadonlyModifier?: boolean;
+  jsDoc?: OverrideJsDocOptions;
```

Comment:
normalized does not accept `undefined`.
It also aligns the naming convention with other override options.

```suggestion
  jsDoc: OverrideJsDocOptions;
```

You also need to define the normalized type separately.
For example, refer to hono's override.

- https://github.com/orval-labs/orval/blob/master/packages/core/src/types.ts#L470-L475
- https://github.com/orval-labs/orval/blob/master/packages/core/src/types.ts#L551-L556